src/kernel/src/fs/null/vnode.rs

```diff
@@ -0,0 +1,80 @@
+use super::NullFs;
+use crate::{
+    errno::{Errno, EROFS},
+    fs::{perm::Access, MountFlags, OpenFlags, VFile, Vnode, VnodeType, VopVector},
+    process::VThread,
+};
+use std::{num::NonZeroI32, sync::Arc};
+use thiserror::Error;
+
+#[allow(dead_code)]
+pub(super) static VNODE_OPS: VopVector = VopVector {
+    default: None,
+    access: Some(access),
+    accessx: Some(access),
+    getattr: None,
```

Comment:
This one is not null on the PS4 so better to add it with `todo` instead of leaving this.